rdr_service/model/participant_summary.py

```diff
@@ -1386,6 +1386,14 @@ def enrollmentSiteId(cls):
     lastModified = Column("last_modified", UTCDateTime6)
     """UTC timestamp of the last time the participant summary was modified"""
 
+    #Placeholder fields for deprecated columns
+    physicalMeasurementsStatus = None
+    physicalMeasurementsTime = None
+    physicalMeasurementsFinalizedTime = None
+    physicalMeasurementsCreatedSite = None
+    physicalMeasurementsFinalizedSite = None
+    physicalMeasurementsCollectType = None
+
```

Comment:
Not sure if we need these here, in the dao since you are appending the key/values onto the `result` dict anyway.

Reply:
I agree, I think they could be taken out. I'm going ahead to approve the PR, but taking them out may be a good idea.

Reply:
I took them out locally and didn't see any issues while I was testing so I'm updating the PR.